.circleci/config.yml

```diff
@@ -932,56 +880,78 @@ jobs:
+        default: ""
+      packages:
+        description: List of packages to test
+        type: string
+        default: |
+          op-batcher
+          op-chain-ops
+          op-node
+          op-proposer
+          op-challenger
+          op-dispute-mon
+          op-conductor
+          op-program
+          op-service
+          op-supervisor
+          op-deployer
+          op-e2e/system
+          op-e2e/e2eutils
+          op-e2e/opgeth
+          op-e2e/interop
+          op-e2e/actions
+          op-e2e/faultproofs
     machine: true
     resource_class: <<parameters.resource_class>>
     steps:
       - checkout
       - attach_workspace:
-          at: ./tmp/workspace
-      - run:
-          name: Load devnet-allocs and artifacts
-          command: |
-            mkdir -p .devnet
-            cp -r ./tmp/workspace/.devnet* .
-            cp -r ./tmp/workspace/packages/contracts-bedrock/forge-artifacts packages/contracts-bedrock/forge-artifacts
-            cp ./tmp/workspace/packages/contracts-bedrock/deploy-config/devnetL1.json packages/contracts-bedrock/deploy-config/devnetL1.json
-            cp -r ./tmp/workspace/packages/contracts-bedrock/deployments/devnetL1 packages/contracts-bedrock/deployments/devnetL1
-      - run:
-          name: print go's available MIPS targets
-          command: go tool dist list | grep mips
+          at: "."
       - run:
           name: run tests
           no_output_timeout: <<parameters.no_output_timeout>>
           command: |
-            mkdir -p ./tmp/testlogs
-            mkdir -p ./tmp/test-results
+            mkdir -p ./tmp/test-results && mkdir -p ./tmp/testlogs
+            
+            cd op-e2e && make pre-test && cd ..
+
+            packages=(
+              <<parameters.packages>>
+            )
+            formatted_packages=""
+            for package in "${packages[@]}"; do
+              formatted_packages="$formatted_packages ./$package/..."
+            done
 
-            # The below env var gets overridden when running make test-cannon, but we
-            # need to explicitly set it here to prevent Cannon from running when we don't
-            # want it to.
             export OP_E2E_CANNON_ENABLED="false"
-            export OP_E2E_SKIP_SLOW_TEST=<<parameters.skip_slow_tests>>
-            # Note: We don't use circle CI test splits because we need to split by test name, not by package. There is an additional
-            # constraint that gotestsum does not currently (nor likely will) accept files from different packages when building.
-            JUNIT_FILE=../tmp/test-results/<<parameters.module>>_<<parameters.target>>.xml JSON_LOG_FILE=../tmp/testlogs/test.log make <<parameters.target>>
-          working_directory: <<parameters.module>>
+            export OP_E2E_SKIP_SLOW_TEST=true
+            export OP_E2E_USE_HTTP=true
+            export ENABLE_ANVIL=true
+            export SEPOLIA_RPC_URL="https://ci-sepolia-l1-archive.optimism.io"
```

Comment:
I'm not sure, but shouldn't these RPC endpoints be parameters of the job?
Although I guess the overrides could take care of it

Reply:
They're defaults. They can be overridden by the env var.